test/lib/sharness/aggregate-results.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+#
+# Copyright (c) 2008-2012 Git project
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses/ .
```

Comment:
@mappum you vendored sharness src. we can't vendor it because it's GPL. that's why our makefile goes through an install procedure. pls unvendor